image_generator.py

```diff
@@ -0,0 +1,33 @@
+import cv2
+
+
+# returns an tuple with channel values as respectice indices at the specified image pixel
+def get_pixel_channel_values(image, x, y):
+    return image[x, y][0], image[x, y][1], image[x, y][2]
+
+
+# returns the channel value specified for the image pixel
+def get_pixel_channel(image, x, y, channel=0):
+    return image[x, y, channel]
+
+
+# returns 2-dimensional tuple with channel values corresponding to the respective indices of the image
+def get_image_channel(image, channel=0):
```

Comment:
For instance, this could be just `image[:, :, channel]`.

Reply:
CV color detection and CV ball detection won't be needing this file, so we will be removing this file from our spec.